src/main/java/com/parasoft/findings/bamboo/ReportCollector.java

```diff
@@ -77,6 +77,8 @@ public TestCollectionResult collect(File file) throws Exception {
 
     private InputStream getInputStream(File file, String xslFile)
             throws FileNotFoundException, TransformerException {
+        // Resolve the conflict of TransformerFactory and Xalan Dependency
+        System.setProperty("javax.xml.transform.TransformerFactory", "com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl");
         TransformerFactory tFactory = TransformerFactory.newInstance();
         // https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#transformerfactory
         tFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, ""); //$NON-NLS-1$
```

Comment:
The error is `TransformerFactory Not supported: http://javax.xml.XMLConstants/property/accessExternalDTD` that is thrown in this line